Tests/test_imagefont.py

```diff
@@ -564,6 +564,7 @@ def loadable_font(
     # catching syntax like errors
     monkeypatch.setattr(sys, "platform", platform)
     if platform == "linux":
+        monkeypatch.setenv("XDG_DATA_HOME", "/home/__pillow__/.local/share")
```

Comment:
```suggestion
        monkeypatch.setenv("XDG_DATA_HOME", os.path.expanduser("~/.local/share"))
```
This seems better to me.